tests/assignments.js

```diff
@@ -0,0 +1,79 @@
+const request = require("supertest");
+const app = require("../server");
+const User = require("../models/User");
+const Assignment = require("../models/AssignmentModel");
+const jwtDecode = require('jwt-decode');
+
+var token;
+var userId;
+var testDate = new Date();
+
+beforeAll(async () => {
+  await request(app)
+    .post("/api/users/register")
+    .send({ 
+      name: "testingUser",
+      email: "user@test.com",
+      password: "testingUser",
+      password2: "testingUser"
+    })
+    .then(response => {
+      expect(response.statusCode).toBe(200);
+    });
+  await request(app)
+    .post("/api/users/login")
+    .send({ 
+      email: "user@test.com",
+      password: "testingUser"
+    })
+    .then(response => {
+      expect(response.statusCode).toBe(200);
+      token = response.body.token
+      userId = jwtDecode(token).id;
+    });
+});
+
+describe("Basic example tests", () => {
+  let assignmentId;
+  test("Create Assignment", () => {
+    return request(app)
+    .post("/api/assignments/")
```

Comment:
Need to pass the jwt as the Authorization header in order to get a response. 

adding `.set({` 'Authorization: token })`